rero_ils/query.py

```diff
@@ -395,7 +395,8 @@ def _default_parser(qstr=None, query_boosting=[]):
         """Default parser that uses the Q() from elasticsearch_dsl."""
         query_type = 'query_string'
         default_operator = 'OR'
-        if request.args.get('simple'):
+        simple = request.args.get('simple')
+        if simple and simple == '1':
```

Comment:
could be replaced by : `if request.args.get('simple') == '1':`

Reply:
done